Catalog app crashes upon clicking auto complete widget

**Describe the bug**
Catalog app crashes upon clicking auto complete widget

**Component**
Catalog app

**To Reproduce**
Steps to reproduce the behavior:
1. Open catalog app
2. Click auto complete

**Expected behavior**
App does not crash

**Screenshots**
NA

**Smartphone (please complete the following information):**
 - Device: Pixel7 Pro

**Additional context**
NA

**Would you like to work on the issue?**
Yes

```
Caused by: java.lang.IllegalArgumentException: Questionnaire item 1 has both initial value(s) and has answerOption. See rule que-11 at https://www.hl7.org/fhir/questionnaire-definitions.html#Questionnaire.item.initial.
```

--- a/catalog/src/main/assets/component_auto_complete.json
+++ b/catalog/src/main/assets/component_auto_complete.json
@@ -25,6 +25,26 @@
           }
         }
       ],
+      "item": [
+        {
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-displayCategory",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "http://hl7.org/fhir/questionnaire-display-category",
+                    "code": "instructions"
+                  }
+                ]
+              }
+            }
+          ],
+          "linkId": "instruction",
+          "text": "Try typing Asthma, Chronic Lung Disease, Depression, Diabetes, Hypertension, High Blood Pressure, or High Cholesterol",
+          "type": "display"
+        }
+      ],
       "repeats": true,
       "linkId": "1",
       "text": "Do you have any existing conditions",
@@ -33,7 +53,8 @@
           "valueCoding": {
             "code": "asthma",
             "display": "Asthma"
-          }
+          },
+          "initialSelected": true
         },
         {
           "valueCoding": {
@@ -71,13 +92,7 @@
             "display": "High Cholesterol"
           }
         }
-      ],
-      "initial": {
-        "valueCoding": {
-          "code": "asthma",
-          "display": "Asthma"
-        }
-      }
+      ]
     }
   ]
 }

